test/fixtures.js

```diff
@@ -16,14 +16,19 @@ export const EMAIL_PASTED = 'foo789@bar.com';
 export const EMAIL_CLEAN = 'baz789@qux.com';
 export const EMAIL_INVALID = 'foobar';
 export const PASSWORD_PWNED = 'password';
-export const PASSWORD_CLEAN = 'sekret';
+export const PASSWORD_CLEAN = 'reallyitsclean';
 export const OPTS_DOM = { domain: DOMAIN };
 export const OPTS_TRUNC = { truncate: true };
 export const OPTS_DOM_TRUNC = { domain: DOMAIN, truncate: true };
-export const OPTS_ISAHASH = { isAHash: true };
-export const RANGE_VALID = '21BD1';
+export const RANGE_PASSWORD_PWNED = '5BAA6';
+export const RANGE_PASSWORD_CLEAN = '01EEC';
 export const RANGE_INVALID = '21BD';
-export const RESPONSE_STR = 'Stuff';
 export const RESPONSE_OBJ = {};
 export const RESPONSE_ARY = [];
 export const RESPONSE_CLEAN = null;
+export const RANGE_PASSWORD_PWNED_RESPONSE = `003D68EB55068C33ACE09247EE4C639306B:3
```

Comment:
Let's add `common-tags` as a dev dependency and format these with `stripIndents` like in [`pwned`](https://github.com/wKovacs64/pwned/blob/6970c5911e6dda356c806fd03ddb1f77b491316c/test/fixtures.js).

Reply:
Very cool!